site/content/en/docs/administration/advanced/upgrade_guide.md

```diff
@@ -56,6 +56,12 @@ To upgrade CVAT, follow these steps:
   docker logs cvat_server -f
   ```
 
+## Upgrade CVAT after v2.25.0
+
+In version 2.25.0, CVAT changed the location where the export cache is stored.
+To clean up the outdated cache, run the following command: `python manage.py exportcachecleanup`.
```

Comment:
```suggestion
To clean up the outdated cache, run the following command: `docker exec -it cvat_server python manage.py exportcachecleanup`.
```

Reply:
how about a case when CVAT is deployed with kubernetes?

Reply:
Both variants are incorrect, need to find a better one.

Reply:
Maybe just add 3 options for simplicity. Organize them in tabs.

Reply:
> Maybe just add 3 options for simplicity. Organize them in tabs.

Sounds good, okay